editor/modes/visual-editor/block-list.js

```diff
@@ -164,7 +172,12 @@ class VisualEditorBlockList extends wp.element.Component {
 
 		return (
 			<div>
-				{ blocksWithInsertionPoint.map( ( uid ) => {
+				{ ! blocks.length && (
+					<button className="editor-visual-editor__placeholder" onClick={ this.appendDefaultBlock }>
```

Comment:
Should this be `onFocus` so it becomes activated on tabbing from the title as well?